src/main/java/io/codelex/test/exercise4/ReverseContent.java

```diff
@@ -0,0 +1,34 @@
+package io.codelex.test.exercise4;
+
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReverseContent {
+
+    public static void main(String[] args) throws IOException {
```

Comment:
Šis strādā par 99%, jo visu izdara pareizi, tikai faila nosaukumu neaprgriež otrādi - bet tas sīkums, vari atstāt kā ir.

Reply:
Paldies, izlaboju!